pkg/util/async/runner.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
You'll need to remove the "All rights reserved" from these - new boilerplate has removed it.